.github/workflows/frontend-lint-burndown.yml

```diff
@@ -17,9 +17,9 @@ jobs:
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
 
-      - name: Install dependencies & inject eslint-plugin-deprecation
```

Comment:
The rules inside this plugin are built-in to [typescript-eslint](https://typescript-eslint.io/)  now. Enabling it adds 1m of runtime and requires that `jsDocParsingMode: 'all'` be set. 

See https://typescript-eslint.io/packages/parser/#jsdocparsingmode

https://www.npmjs.com/package/eslint-plugin-deprecation